pymzml/run.py

```diff
@@ -238,6 +242,67 @@ def _determine_file_encoding(self, path):
             with _open(path, 'rb') as sniffer:
                 return self._guess_encoding(sniffer)
 
+    @staticmethod
+    def _obo_version_validator(version):
+
+        """
+        The obo version should fit file names in the obo folder.
+        However, some software generate mzML with built in obo version string like:
+        '23:06:2017' or even newer version that not in obo folder yet.
+        This is to check obo version and try to fit the best obo version
+        to the obo version in the mzML file.
+
+        :param version: obo version number as string
+        :return:
+        """
+
+        obo_rgx = re.compile(r'(\d\.\d{1,2}\.\d{1,2})(_[rR][cC]\d{0,2})?')
+        obo_years_rgx = re.compile(r'20\d\d')
+        obo_year_version_dct = {
+            2012: '3.40.0', 2013: '3.50.0', 2014: '3.60.0',
+            2015: '3.75.0', 2016: '4.0.1', 2017: '4.1.0',
+            2018: '4.1.10', 2019: '4.1.22',
+        }
```

Comment:
I guess this needs to be maintained in the future for years exceeding 2019 or is line 281/182 sufficient?

Reply:
I think if there are not so many changes in the obo file, then 281/282 is enough.
Since I did not track the development of obo files, I propose this part to fit an obo_version as close as possible.
In my opinion, add a stable version once per year is not that difficult.

Reply:
True, one stable version is just fine for me, just wanted to check back with you